plugins/parquet/parquetembed.hpp

```diff
@@ -108,7 +108,7 @@ struct ArrayBuilderTracker
     unsigned int childrenProcessed = 0;
 
     ArrayBuilderTracker(const char *_nodeName, arrow::ArrayBuilder *_struct, PathNodeType _nodeType, arrow::FieldPath  _nodePath)
-        : nodeName(_nodeName), nodeType(_nodeType), structPtr(_struct), nodePath(_nodePath) { if (nodeType == CPNTDataset) childCount == structPtr->num_children(); }
+        : nodeName(_nodeName), nodeType(_nodeType), structPtr(_struct), nodePath(std::move(_nodePath)) { if (nodeType == CPNTDataset) childCount = structPtr->num_children(); }
```

Comment:
If you are using std::move on _nodePath then it should be an && parameter (and std::move also used on the call)

Also, more normal style would be to have the { etc on separate lines.  e.g.

```
{
  if (nodeType == CPNTDataset)
    childCount = structPtr->num_children();
}

```